priv/beam.json

```diff
@@ -106,15 +104,15 @@
               },
               {
                 "color": "#EAB839",
-                "value": 150000
+                "value": 500000000
```

Comment:
It's difficult to pick a sensible default, consider using a percentage of the memory available on the system instead.

Reply:
I'll create an issue for this one for tracking purposes...but this will have to wait until dashboard templating is supported so that the system memory can be evaluated prior to uploading the dashboard. I think that dashboard templating will begin in the next 2-3 weeks.